services/web/package.json

```diff
@@ -12,6 +12,10 @@
   "dependencies": {
     "@hookform/resolvers": "^3.9.1",
     "@radix-ui/react-accordion": "^1.2.0",
+    "@radix-ui/react-label": "^2.1.0",
```

Comment:
These are all downloaded from shadcn. Investigate if all of them are used. If some of them are not follow shadcn documentation on how to remove them.